cmd/agent/agent.go

```diff
@@ -47,7 +52,13 @@ func NewAgentCmd() *cobra.Command {
 
 	startCmd.Flags().StringVar(&sshAddress, "ssh-address", "", "The address to which the trento-agent should be reachable for ssh connection by the runner for check execution.")
 
-	startCmd.Flags().IntVarP(&discoveryPeriod, "discovery-period", "", 10, "Discovery mechanism loop period in seconds")
+	startCmd.Flags().IntVarP(&clusterDiscoveryPeriod, "cluster-discovery-period", "", 10, "Cluster discovery mechanism loop period in seconds")
+	startCmd.Flags().IntVarP(&sapSystemDiscoveryPeriod, "sapsystem-discovery-period", "", 10, "SAP systems discovery mechanism loop period in seconds")
+	startCmd.Flags().IntVarP(&cloudDiscoveryPeriod, "cloud-discovery-period", "", 10, "Cloud discovery mechanism loop period in seconds")
+	startCmd.Flags().IntVarP(&hostDiscoveryPeriod, "host-discovery-period", "", 10, "Host discovery mechanism loop period in seconds")
+	startCmd.Flags().IntVarP(&subscriptionDiscoveryPeriod, "subscription-discovery-period", "", 900, "Subscription discovery mechanism loop period in seconds")
+
+	startCmd.Flags().MarkHidden("subscription-discovery-period")
```

Comment:
Why is this flag hidden if we have validation later?

Reply:
Yes, giving it another thought, the fact that it's allows us to put a lower min value in the validation